integration/testdata/cis-1.6/job-master.data

```diff
@@ -70,10 +70,7 @@
 [PASS] 1.4.2 Ensure that the --bind-address argument is set to 127.0.0.1 (Automated)
 
 == Remediations master ==
-1.1.9 Run the below command (based on the file location on your system) on the master node.
-For example,
-chmod 644 <path/to/cni/files>
-
+1.1.9 audit test did not run: No tests defined
```

Comment:
Why during tests kube-bench doesn't see audit tests?

Reply:
From a quick look I can't tell since the yaml seems fine, this error means there is no test item object for this check, what means somewhere in the process there was an issue with this test, could you run it with the -v 3 flag for more details? (recommend also to use the --check="1.1.9" flag to only check for this test)

Reply:
How can i reproduce it with v3?

Reply:
Because on my cluster and also on local kind cluster everything works

Reply:
@yoavrotems ping

Reply:
Ok I looked farther and the yaml is indeed valid yaml but missing the "test:" item